driver/src/main/java/org/neo4j/driver/ExecuteQueryConfig.java

```diff
@@ -27,15 +27,15 @@
 import org.neo4j.driver.util.Experimental;
 
 /**
- * Query configuration used by {@link Driver#queryTask(String)} and its variants.
+ * Query configuration used by {@link Driver#executeQueryTemplate(String)} and its variants.
  * @since 5.5
  */
 @Experimental
-public final class QueryConfig implements Serializable {
+public final class ExecuteQueryConfig implements Serializable {
```

Comment:
Isn't this one named `QueryConfig` in the ADR?

Reply:
oops, yes, went a bit overboard, will revert that change